pkg/anonymization/anonymizer.go

```diff
@@ -108,14 +108,14 @@ func NewAnonymizerFromConfigClient(
 	configClient configv1client.ConfigV1Interface,
 	networkClient networkv1client.NetworkV1Interface,
 	configurator configobserver.Interface,
-	dataPolicy v1alpha1.DataPolicy,
+	dataPolicy []v1alpha2.DataPolicyOption,
```

Comment:
If DataPolicyOption is now a collection of items, wouldn't be better to change this field to `dataPolicies` or `dataPolicyOptions`?